app/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  arrowParens: 'avoid',
+  bracketSameLine: true,
+  bracketSpacing: false,
+  singleQuote: true,
+  trailingComma: 'all',
```

Comment:
Is this the default, or is this what you're proposing? I prefer `bracketSpacing` to be true...

Reply:
I'll change this to `true`, it was `false` by default from the RN boilerplate.